lom/src/lib/lomcommon/helper.go

```diff
@@ -151,6 +152,100 @@ func LogDebug(s string, a ...interface{})  {
     LogMessage(syslog.LOG_DEBUG, s, a...)
 }
 
+/*
```

Comment:
Can you add usage example like sequence of calls ?

Reply:
Comments added.